scripts/build.sh

```diff
@@ -97,9 +97,22 @@ rm -rf ./proto/.git
 # Primary use is TLS between Broker-CLI and Broker Daemon
 #
 #############################################
-KEY_PATH="./certs/broker-rpc-tls.key"
-CERT_PATH="./certs/broker-rpc-tls.cert"
-CSR_PATH="./certs/broker-rpc-csr.csr"
+
+DIRECTORY=~/.sparkswap
+if [ ! -d "$DIRECTORY" ]; then
```

Comment:
`mkdir -p` won't create the directory if it doesn't exist (and I think it is recursive), so why not just have this and the below like:

```
echo "Creating directories $DIRECTORY and $DIRECTORY/secure"
mkdir -p $DIRECTORY/secure
```